tensorflow/contrib/layers/__init__.py

```diff
@@ -28,6 +28,7 @@
 @@flatten
 @@fully_connected
 @@layer_norm
+@@layer_norm_fused
```

Comment:
I think it is fine to put into layers. contrib/layers is highly experimental, adding functions there shouldn't be a problem.